fe/fe-core/src/test/java/com/starrocks/alter/AlterTest.java

```diff
@@ -159,7 +157,7 @@ private static void createTable(String sql) throws Exception {
     }
 
     private static void alterTable(String sql, boolean expectedException) throws Exception {
```

Comment:
Don't modify UtFrameUtils.parseStmtWithNewParser, you can add an alterTableWithNewParser function